driver-mct/cime_config/config_component_e3sm.xml

```diff
@@ -389,9 +391,11 @@
       <value compset="_MPASO">48</value>
       <value compset="_DLND.*_MALI">1</value>
```

Comment:
If the base coupling interval for MALI is set to 1 year, does the above value for "_DLND.*_MALI" compsets also need to be updated? Even if only for consistency? Or, does this compset really only couple w/ the atmos. once per year?